### PR TITLE
Docs: remove deprecated "Debug Dreamview" document

### PR DESCRIPTION
**Pull request by storypku**

### CODE REVIEW

File: docs/FAQs/CyberRT_FAQs.md [hunk 1]
```diff
@@ -1 +1,2 @@
-Please refer to the  [Apollo Cyber RT FAQ](https://github.com/ApolloAuto/apollo/tree/master/docs/cyber/CyberRT_FAQs.md) under docs/cyber directory.
+Please refer to [Apollo CyberRT FAQ](../cyber/CyberRT_FAQs.md) under the
+`docs/cyber` directory.
```
Comment (storypku) [inline docs/FAQs/CyberRT_FAQs.md:2]: Use relative path.

File: docs/FAQs/Dreamview_FAQs.md [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
 `bash scripts/bootstrap.sh`.
 
 So if you can not access Dreamview, please check:
 
-- Make sure you have dreamview process running correctly. In the latest version,
-  `bash scripts/bootstrap.sh` will report `dreamview: ERROR (spawn error)` if
-  dreamview fails to start. For early version, please check with command:
-  `supervisorctl status dreamview` or `ps aux | grep dreamview`. If dreamview is
-  not running, please refer to
-  [How to Debug a Dreamview Start Problem](../howto/how_to_debug_dreamview_start_problem.md)
```
Comment (storypku) [inline docs/FAQs/Dreamview_FAQs.md:19]: Remove obsolete document link.

File: docs/FAQs/Dreamview_FAQs.md [hunk 3]
```diff
@@ -112,144 +108,6 @@ each time the sim_control is restarted. ![](images/sim_control_2.0.png)
...
-```
-
-2. Compile pcl and copy the pcl library files to `/usr/local/lib`:
-
-Inside docker:
-
-```
-(to keep pcl in host, we save pcl under /apollo)
-cd /apollo
-git clone https://github.com/PointCloudLibrary/pcl.git
```
Comment (storypku) [inline docs/FAQs/Dreamview_FAQs.md:195]: We are based on pcl 1.10.1 now. And the `-mavx` problem should be avoided using the pcl patch when docker build.

File: docs/howto/README.md [hunk 4]
```diff
@@ -58,19 +56,13 @@
...
 - [How to Leverage Apollo's RTK mode](how_to_leverage_rtk_mode.md)
 - [How to run the Map Data Verification Tool](how_to_run_map_verification_tool.md)
-- [How to run offline perception visualizer](how_to_run_offline_perception_visualizer.md)
-- [How to run offline sequential obstacle perception visualizer](how_to_run_offline_sequential_obstacle_perception_visualizer.md)
 - [How to run perception module on your local computer](how_to_run_perception_module_on_your_local_computer.md)
 - [How to tune control parameters](how_to_tune_control_parameters.md)
 - [How to update vehicle calibration](how_to_update_vehicle_calibration.md)
 - [How to do performance profiling](how_to_do_performance_profiling.md)
-- [How to run Offline Perception Visualizer for Apollo 2.5](how_to_run_apollo_2.5_perception_visualizer.md)
-- [How to use Common Cyber Commands - Basic guide](how_to_use_cyber_commands.md)
```
Comment (storypku) [inline docs/howto/README.md:68]: Cyber commands are already provided in docs/cyber. The rather rough introduction here didn't help much.

File: docs/howto/README.md [hunk 5]
```diff
@@ -58,19 +56,13 @@
 ## Helper tools and optimization
 - [How to Leverage Apollo's RTK mode](how_to_leverage_rtk_mode.md)
 - [How to run the Map Data Verification Tool](how_to_run_map_verification_tool.md)
-- [How to run offline perception visualizer](how_to_run_offline_perception_visualizer.md)
-- [How to run offline sequential obstacle perception visualizer](how_to_run_offline_sequential_obstacle_perception_visualizer.md)
 - [How to run perception module on your local computer](how_to_run_perception_module_on_your_local_computer.md)
 - [How to tune control parameters](how_to_tune_control_parameters.md)
 - [How to update vehicle calibration](how_to_update_vehicle_calibration.md)
 - [How to do performance profiling](how_to_do_performance_profiling.md)
-- [How to run Offline Perception Visualizer for Apollo 2.5](how_to_run_apollo_2.5_perception_visualizer.md)
```
Comment (storypku) [inline docs/howto/README.md:67]: Deprecated doc can only lead to misleading.

**Review by lfcarol — APPROVED** [review]

**Comment by storypku** [comment]
Please help review again @lfcarol @changsh726 . The original PR was blocked as License/CLA status not known.

**Review by changsh726 — APPROVED** [review]

**Pull request merged**